server/job.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package main
+
+import (
+	"time"
+
+	"github.com/mattermost/mattermost/server/public/pluginapi/cluster"
+	"github.com/pkg/errors"
+)
+
+const (
+	jobKeyStartSurveyJob = "job_start_survey"
+
+	// TODO - update this to 15 minutes once ready for production
+	startSurveyJonInterval = 15 * time.Second
+)
+
+func (p *Plugin) startScheduledJobs() error {
+	if err := p.startSurveyStartJob(); err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
This is just a wrapper over `startSurveyStartJob`. Do we want to keep it separate to add more stuff in the future?

Also, in its current shape, even if we want to keep different names, we can do just
```suggestion
func (p *Plugin) startScheduledJobs() error {
	return p.startSurveyStartJob()
}
```

Reply:
I kept it this way because I thought we'd need two jobs - one to start a survey and one to end, but a single job is sufficient and better for it, so `startScheduledJobs` is unnecessary now. I've removed it now and can add it later when there is an actual need for it.